### PR TITLE
Remove Raise.With<TEventhandler>

### LINKED ISSUE #1152: Remove Raise.With<TEventHandler> in 5.0.0
From https://github.com/FakeItEasy/FakeItEasy/pull/1150/files#r123444441.
It's been superseded by `Raise.FreeForm.With`.

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #1152.

### CODE REVIEW

File: docs/raising-events.md [hunk 1]
```diff
@@ -77,11 +77,11 @@ fake.CustomEvent += Raise.FreeForm.With(fake, sampleCustomEventArgs);
 Due to language limitations, `Raise.Freeform.With` does not work in VB.NET, and it uses late binding, so you need a reference to the `Microsoft.CSharp` assembly in order to use it.
 
 ### From VB.NET
-To raise a free-form event from VB.NET, you must use `Raise.FreeForm<TEventHandler>.With`:
+To raise a free-form event from VB.NET, you must use `Raise.FreeForm(Of TEventHandler).With`:
 
 ```csharp
```
Comment (thomaslevesque) [inline docs/raising-events.md]: `vbnet`, not `csharp` ;)

File: docs/raising-events.md [hunk 2]
```diff
@@ -77,11 +77,11 @@ fake.CustomEvent += Raise.FreeForm.With(fake, sampleCustomEventArgs);
 Due to language limitations, `Raise.Freeform.With` does not work in VB.NET, and it uses late binding, so you need a reference to the `Microsoft.CSharp` assembly in order to use it.
 
 ### From VB.NET
-To raise a free-form event from VB.NET, you must use `Raise.FreeForm<TEventHandler>.With`:
+To raise a free-form event from VB.NET, you must use `Raise.FreeForm(Of TEventHandler).With`:
```
Comment (thomaslevesque) [inline docs/raising-events.md]: Not sure how we missed this :/

File: docs/raising-events.md (hunk 1 above)
Comment (blairconrad) [reply docs/raising-events.md]: Ha. Oops. Thanks.

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque. Squashed.

**Comment by blairconrad** [comment]
… and rebased.

**Comment by thomaslevesque** [comment]
Thanks @blairconrad!

**Pull request merged**